contracts/oracle/implementations/arbitrum/EUR/OracleSTEURETHChainlinkArbitrum.sol

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.12;
+
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+import "../../../BaseOracleChainlinkMultiTwoFeeds.sol";
+import "../../../../interfaces/external/IERC4626.sol";
+
+/// @title OracleSTEURETHChainlinkArbitrum
+/// @author Angle Labs, Inc.
+/// @notice Gives the price of stEUR in ETH in base 18
+contract OracleSTEURETHChainlinkArbitrum is BaseOracleChainlinkMultiTwoFeeds, AggregatorV3Interface {
```

Comment:
But we don't implement most function of AggregatorV3Interface

Reply:
we do, check the interface how it's defined